CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # CHANGELOG
 
+## 1.25.2.dev0 (2022-05-13)
+
+MLflow 1.25.2.dev0 includes several major features and improvements:
+
+Features:
+
+- Add endpoint changes to the deployment CLI. (#5731, @trangevi)
+- Dev env setup script (#5717, @BenWilson2)
+- Update mlflow.set_tracking_uri to support pathlib.Path (#5820) (#5824, @cacharle)
+- Scoring server timeout #5381 (#5663, @sniafas)
+- [FIX-5789] Added sticky header to experiment run list (#5818, @hubertzub-db)
+- Added mechanism to delete artifacts from GCS repository using MLFlow garbage collection (#5811, @aditya-iyengar-rtl-de)
+- Pass in the `pos_label` parameter to `eval_and_log_metrics` (#5807, @yxiong)
+- Show latest, minimum and maximum metric values on metric page (#5574, @adamreeve)
+- Autologging of input example and signature for pyspark ml (#5719, @bali0019)
+- Add virtualenv support for `mlflow models docker-build` (#5728, @harupy)
+- feat: support using wildcards in log_model_allowlist (#5723, @serena-ruan)
+- Add virtualenv support for MLflow projects (#5631, @harupy)
+- Document virtualenv suppport for MLflow projects (#5727, @harupy)
+- Make `pyfunc.spark_udf` support virtualenv (#5676, @WeichenXu123)
+- Added autologging of input example and signature for tf.estimator and keras fit_generator (#5510, @bali0019)
+- Adds JSON Schema Validation (#5458, @mrkaye97)
+- Update UI (#5711, @harupy)
```

Comment:
@harupy How to refine the description of the commit ?

Reply:
#5711 didn't add new features or fix bugs. We can move this the small bug fixes and doc updates section.